pkg/chartmuseum/handlers.go

```diff
@@ -29,6 +29,16 @@ type (
 	filenameFromContentFn func([]byte) (string, error)
 )
 
+func (server *Server) getHealthCheck(c *gin.Context) {
```

Comment:
I am thinking that this route should actually just return a 200. Querying changes to index.yaml may mean ALOT of API requests to the cloud provider (if in use), depending on the frequency of the health check.

Can we create a `healthCheckResponse` up above, such as
```
healthCheckResponse = gin.H{"healthy": true}
```

Then this can become just
```
func (server *Server) getHealthCheck(c *gin.Context) {
	c.JSON(200, healthCheckResponse)
}
```
It'd be nice if health check could return the version/revision of the chartmuseum build too, but unfortunately that info isnt currently being passed from cmd/chartmuseum to pkg/chartmuseum. Then we could do
```
healthCheckResponse = gin.H{"version": version, "revision": revision}
```